test/demo/start

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+
+from drenv import kubectl
+
+if len(sys.argv) != 2:
+    print(f"Usage: {sys.argv[0]} cluster")
+    sys.exit(1)
+
+cluster = sys.argv[1]
+
+print("Deploying demo")
+kubectl.apply("--filename", "demo/deployment.yaml", context=cluster)
```

Comment:
Please use "deployment.yaml" - it will work since the script should change the current working directory to the script directory.

Same for all other files.